webpack/assets/javascripts/react_app/components/common/EmptyState/DefaultEmptyState.js

```diff
@@ -11,31 +12,18 @@ const DefaultEmptyState = props => {
     iconType,
     header,
     description,
-    documentation: {
-      url,
-      label = __('For more information please see'),
-      buttonLabel = __('Documentation'),
-    } = {},
+    documentation,
     action,
     secondaryActions,
   } = props;
 
-  const documentationBlock = url ? (
-    <React.Fragment>
-      {label}{' '}
-      <a href={url} target="_blank" rel="noopener noreferrer">
-        {buttonLabel}
-      </a>
-    </React.Fragment>
-  ) : null;
-
   return (
     <EmptyStatePattern
       icon={icon}
       iconType={iconType}
       header={header}
       description={description}
-      documentation={documentationBlock}
+      documentation={documentation}
```

Comment:
can you change the `proptype` of this prop in `EmptyStatePattern` ?
since it's no more an element but an object with a specific shape, e.g:
```js
PropTypes.shape({
  url: PropTypes.string,
  label: PropTypes.string,
  buttonLabel: PropTypes.string,
}),
```